jobs/buildhub/utils.py

```diff
@@ -109,10 +109,24 @@ def is_release_filename(product, filename):
         product = "firefox"
     match_filename = filename.replace(' ', '-').lower()
     re_filename = re.compile("{}-(.+)({})$".format(product, FILE_EXTENSIONS))
-    re_exclude = re.compile(".+(sdk|tests|crashreporter|stub)")
+    re_exclude = re.compile(".+(sdk|tests|crashreporter|stub|gtk2.+xft|-source)")
     return re_filename.match(match_filename) and not re_exclude.match(match_filename)
 
 
+def is_release_url(product, url):
```

Comment:
Since it is used in the same place as `is_release_filename()` and takes the same argument, I believe we can combine both